src/components/Table.tsx

```diff
@@ -24,6 +24,14 @@ type Props = {
   tableHeight?: string;
 };
 
+export const ClickableRowPropsConfiguration = {
+  style: {
+    cursor: 'pointer',
+    transition: 'background-color 0.8s',
+  },
+  _hover: { bg: theme.colors.teachHub.gray },
```

Comment:
Esto del `theme` está pensado para ser invocado como string, sin necesidad de importar `theme`

```suggestion
  _hover: { bg: 'teachHub.gray' },
```

https://chakra-ui.com/docs/styled-system/customize-theme#customizing-theme-tokens